### DISCUSSION TITLE
I would like to know some questions about training

### DISCUSSION THREAD

**Discussion by Nerva05251228** [NVIDIA-NeMo/NeMo#7298] [Q&A]
Hello everyone!
I am a novice in ASR and would like to know if I want to train a model for Chinese dialects (such as Cantonese speech recognition). I have a few questions to understand:

1. Should I use a pre trained model for Mandarin? Because I am worried that the pronunciation of Mandarin and Cantonese may not be similar, resulting in training failure.

2. How many voice segments should I select, what should be the duration of each segment, and should the training rounds be as good as possible?

3. How should the data from the training set and the test set be allocated? For example, if I have ten voice segments, is it reasonable for me to allocate seven as a training set and three as a testing set? Or should I use a 1:1 distribution method?

Thank you for your reply! I hope to get some help as soon as possible!

**Comment by titu1994** [comment]
@fayejf
